raster/src/main/scala/geotrellis/raster/render/shared/Renderer.scala

```diff
@@ -14,8 +14,10 @@
  * limitations under the License.
  */
 
-package geotrellis.raster.render.png
+package geotrellis.raster.render.shared
```

Comment:
I'd suggest, instead of having a "shared" package, that this just go in `geotrellis.raster.render`